code/game/gamemodes/blob/overmind.dm

```diff
@@ -91,10 +91,13 @@ GLOBAL_LIST_EMPTY(blob_nodes)
 
 		if(!Ablob.blob_allowed)
 			continue
-
-		playsound(L, 'sound/effects/splat.ogg', 50, 1)
-		L.death()
-		new/mob/living/simple_animal/hostile/blob/blobspore(T)
+		
+		if(!istype(L, /mob/living/simple_animal/hostile/blob))
+			playsound(L, 'sound/effects/splat.ogg', 50, 1)
+			L.death()
+			new/mob/living/simple_animal/hostile/blob/blobspore(T)
+		else
+			L.health = L.maxHealth
```

Comment:
use the proper heal procs, since this won't fully heal them, only make them appear healed until they recalculate health.